src/js/tech/tech.js

```diff
@@ -813,6 +813,8 @@ Tech.withSourceHandlers = function(_Tech){
     // than clear all of our current tracks
     if (this.currentSource_) {
       this.clearTracks(['audio', 'video']);
```

Comment:
I'm not sure I entirely remember but I think it had to do with saving keeping Audio/Video Tracks that were added before we had a currentSource